packages/tooling/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.ts

```diff
@@ -96,12 +97,19 @@ export class HTMLRenderLayerNavigation extends HTMLRenderLayer {
         return new OverlayPosition(pos.top, pos.left, pos.width, pos.height);
     }
 
+    private getTitleForDictionaryId(dataDictionaryId): string | null {
+        if (this.schemaDictionary && this.dataDictionary) {
+            if (this.dataDictionary[0][dataDictionaryId].data[dataSetName])
+                return this.dataDictionary[0][dataDictionaryId].data[dataSetName];
+            else
```

Comment:
Missing braces around the else statement

Reply:
No longer relevant.